src/Appwrite/Resque/Worker.php

```diff
@@ -165,85 +166,28 @@ public static function error(callable $callback): void
      */
     protected function getProjectDB(string $projectId): Database
     {
-        $consoleDB = $this->getConsoleDB();
-
-        if ($projectId === 'console') {
-            return $consoleDB;
+        global $register;
```

Comment:
This is an awful pattern we use. I wonder if we can have same dependency injection for both http, realtime, queue and cli, and then use utopia/platform to set it in a consistent way for all platforms.